java/client/src/main/java/glide/api/commands/HashBaseCommands.java

```diff
@@ -65,6 +65,21 @@ public interface HashBaseCommands {
      */
     CompletableFuture<Long> hdel(String key, String[] fields);
 
+    /**
+     * Returns all values in the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hvals/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @return An <code>array</code> of values in the hash, or an <code>empty array</code> when the
+     *     key does not exist.
+     * @example
+     *     <pre>{@code
+     * String[] values = client.hdel("myHash").get();
```

Comment:
I think this is the wrong command. Also, would be good to set some values first like they do in the official documentation for Redis to show where the values are coming from.

Reply:
Fixed command name. Copied examples from Node and Py.